### PR TITLE
(SERVER-1741) Update tk-ws-jetty9 to 2.0

### PR DESCRIPTION
**Pull request by adrienthebo**

### CODE REVIEW

**Comment by rlinehan** [comment]
I think this should be server-1741...

**Comment by adrienthebo** [comment]
@rlinehan 💩 

I'll fix that.

File: test/integration/puppetlabs/services/legacy_routes/legacy_routes_test.clj [hunk 1]
```diff
@@ -290,7 +290,7 @@
                  "1.2.3")))
      (let [node-response (http-get "/puppet/v3/node/localhost?environment=production")]
        (is (= 200 (:status node-response)))
-       (is (= "request routed to: /puppet/v3/node/localhost")))))
+       (is (= "request routed to: /puppet/v3/node/localhost" (:body node-response))))))
```
Comment (rlinehan) [inline test/integration/puppetlabs/services/legacy_routes/legacy_routes_test.clj]: nice

Comment (camlow325) [inline test/integration/puppetlabs/services/legacy_routes/legacy_routes_test.clj]: d'oh

**Pull request merged**